README.md

```diff
@@ -158,39 +169,33 @@ func main() {
 
 ```
 
-## Additional resources ##########################################################
+## Contributing 
+
+Please feel free to submit issues, fork the repository and send pull requests!
+
+When submitting an issue, we ask that you please include a complete test function that demonstrates the issue.
+
+## Additional resources
 
 Additional resources are available at:
 
-* [Package documentation](https://pkg.go.dev/github.com/cloudinary/cloudinary-go)
+* [Package reference documentation](https://pkg.go.dev/github.com/cloudinary/cloudinary-go)
+* [SDK Documentation](https://cloudinary.com/documentation/go_integration)
+* [Upload API documentation](https://cloudinary.com/documentation/upload_images)
 * [Website](https://cloudinary.com)
 * [Interactive demo](https://demo.cloudinary.com/default)
 * [Knowledge Base](https://support.cloudinary.com/hc/en-us)
-* [Documentation](https://cloudinary.com/documentation)
-* [Upload API documentation](https://cloudinary.com/documentation/upload_images)
-
-## Support
 
-You can [open an issue through GitHub](https://github.com/cloudinary/cloudinary-go/issues).
-
-Stay tuned for updates, tips and tutorials: [Blog](https://cloudinary.com/blog)
-, [Twitter](https://twitter.com/cloudinary), [Facebook](https://www.facebook.com/Cloudinary).
-
-## Join the Community ###########################################################
+## Community and Support
 
 Impact the product, hear updates, test drive new features and more!
 Join [here](https://www.facebook.com/groups/CloudinaryCommunity).
 
-## Staying up to date  ##########################################################
-
-To update Cloudinary Go SDK to the latest version, use `go get -u github.com/cloudinary/cloudinary-go`.
-
-## Contributing  ################################################################
-
-Please feel free to submit issues, fork the repository and send pull requests!
+You can [open an issue through GitHub](https://github.com/cloudinary/cloudinary-go/issues).
```

Comment:
Are we intentionally not also inviting contributions (since we currently don't have a dedicated contributions.md page)?

Reply:
@const-cloudinary - Thoughts on this?

Reply:
@jackieros @njb90 We do invite them. I thought I added contributions.md, I'll find a good template and will add. We should also update it here

Reply:
@const-cloudinary - Sounds good. In that case, here we should just mention the option & link to the contributions page.

Reply:
@jackieros , @njb90 Added `CONTRIBUTING.md` file